### PR TITLE
[AL-60][An] Uplift user profile template

### PR DESCRIPTION
**Pull request by buigiaanfb1**
## Jira link

[AL-44](https://capstone-alumni.atlassian.net/jira/software/c/projects/AL/boards/1?modal=detail&selectedIssue=AL-44)

## Screenshot/Record (FE)
Screen Shot 2023-01-17 at 03 40 29

### CODE REVIEW

**Review by congtrung6391 — COMMENTED** [review]
Profile should be a seperate module

File: src/app/(nowrap_layout)/(session)/profile/page.tsx [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+    paddingRight: theme.spacing(3)
+  }
+}));
+
+// ----------------------------------------------------------------------
+
+export default function UserProfile() {
+  const [currentTab, setCurrentTab] = useState('profile');
+  const [findFriends, setFindFriends] = useState('');
+  const myProfile = {
```
Comment (congtrung6391) [inline src/app/(nowrap_layout)/(session)/profile/page.tsx:49]: mock data should be in seperate file, you could create a folder `__mockData__` in the same module.

**Review by congtrung6391 — COMMENTED** [review]
1. Move the page to (authorized_layout)
2. Move profile component to `profiles` module
3. We will temporary use `ProfileAbout` to show `information`
4. Create edit profile page

**Comment by pvnamGit** [comment]
AL-44 is not a task, it is an epic
My suggestion: Close this PR, create the new task belong to epic AL-44

**Comment by buigiaanfb1** [comment]
> Profile should be a seperate module

Profile will move to authorized app directory once it is done! Thanks Trung

**Comment by buigiaanfb1** [comment]
> 1. Move the page to (authorized_layout)
> 2. Move profile component to `profiles` module
> 3. We will temporary use `ProfileAbout` to show `information`
> 4. Create stateless edit profile page

Yeah about the point number 2, 

> AL-44 is not a task, it is an epic My suggestion: Close this PR, create the new task belong to epic AL-44

Yeah, I will change the title instead

**Pull request merged**